src/util/invariant.cpp

```diff
@@ -93,6 +102,29 @@ void print_backtrace(
       out << '\n' << std::flush;
     }
 
+#elif defined(_WIN32)
+
+  void *stack[50];
+  HANDLE process = GetCurrentProcess();
+
+  SymInitialize(process, NULL, TRUE);
+
+  auto number_of_frames = CaptureStackBackTrace(0, 50, stack, NULL);
```

Comment:
Use a sizeof or a define for 50 to avoid the obvious mistake when someone wants more frames

Reply:
Now using sizeof.